docs/Dashboard.md

```diff
@@ -22,6 +22,23 @@ Quickly manage pending payouts, view recent transactions, and overview outstandi
 
 ![BTCPay Server Navigation](./img/dashboard/tx-activity-view.jpg "BTCPay Server Navigation")
 
+### Lightning Balance
+This will show your available balance in `Lightning Channels` and your `On-Chain` Bitcoin balance. 
```

Comment:
```suggestion

This will show the available balances for your Lightning node. 
Please note that the on-chain balance refers to the wallet of your store's Lightning node, not the store's general on-chain wallet.

```